### ISSUE TITLE
Build bot to tweet events hours before they happen

### ISSUE THREAD

**Issue by ericlathrop** [louisvilletech/louisvilletech.org#9]

**Comment by LilacLlama** [comment]
Per channel, do you think? Or one particular channel for all?

**Comment by ericlathrop** [comment]
I was kinda thinking both.

**Comment by justcarver** [comment]
The issue says tweet bot, but are we wanting one that will post into the Slack channel too? Just a bit confused. (Started taking a shot at a twitter bot).  Just not sure how we determine which other channels for which groups, I would assume general.

**Comment by ericlathrop** [comment]
@justcarver Yes, that should be a separate issue that hasn't been created yet. See issue #3.

**Comment by ericlathrop** [comment]
@justcarver I created the Slack bot issue as #11.
